app/assets/stylesheets/osem.css.scss

```diff
@@ -71,4 +71,14 @@ p.comment-body {
 
 #account-already {
   font-size: 0.6em;
-}
\ No newline at end of file
+}
+
+/* comments views pane: use padding-bottom before next comment box */
+.panel.panel-default .panel-body .box {
```

Comment:
box is a very bad class name and will likely clash with other things in the future...